### ISSUE TITLE
Retire deprecated features: 2018-05

### ISSUE THREAD

**Issue by dustingetz** [hyperfiddle/hyperfiddle#230]
- [x] :hypercrud.browser/cell is unused (superseded by fiddle/renderer `((:cell ctx') [true 0 :fiddle/query] ctx')` - this interface should be improved to accept fn parameters)

**Comment by dustingetz** [comment]
- [ ] migrate !cell to !field (blocked on #285)

**Comment by dustingetz** [comment]
- [ ] archive `:link/ident`
